configure.ac

```diff
@@ -1951,8 +1951,8 @@ if test $enable_fuzz = "no"; then
     echo "  with test       = $use_tests"
 else
     echo "  with test       = not building test_bitcoin because fuzzing is enabled"
-    echo "    with fuzz     = $enable_fuzz"
 fi
+echo "  with fuzz       = $enable_fuzz_binary"
```

Comment:
Just "with fuzz = yes" is still confusing to me. For someone it could be unclear that this message is related to the `--enable-fuzz-binary` flag. Maybe add "binary" word?

Reply:
thx, fixed